benchexec/tools/veriabs.py

```diff
@@ -46,7 +44,7 @@ def name(self):
         return "VeriAbs"
 
     def project_url(self):
-        return "https://www.tcs.com/designing-complex-intelligent-systems"
+        return "https://www.tcs.com"
```

Comment:
Is there a better URL for the VeriAbs project itself? If a user who is interested in VeriAbs clicks on this link, they won't find information about it.

Reply:
we’ve updated the link with the correct URL for accessing the information. We're also working on getting a dedicated project page.